roles/nginx/files/nginx.nomad

```diff
@@ -141,7 +144,9 @@ upstream {{ .Name }} {
 
   keepalive 8;
 }
-
+  {{- if eq .Name (scratch.Get "setProxy") -}}
+proxy_cache_path /var/cache/nginx/{{ .Name }}/ use_temp_path=off keys_zone={{ .Name }}:1m inactive=24h;
+  {{- end -}}
```

Comment:
nitpick: i would generate this next to the server instead of with the upstream so it's a little more readable